common/ethnicities/wc_ethnicities_gnome.txt

```diff
@@ -0,0 +1,401 @@
+	gene_bs_nose_ridge_angle = {
+        1 = { name = nose_ridge_angle_pos    range = { 0.5 1 }    }
+	}
+	gene_bs_nose_ridge_width = {
+        1 = { name = nose_ridge_width_pos    range = { 0 0.8 }    }
+		1 = { name = nose_ridge_width_neg    range = { 0 1 }    }
+	}
+	face_detail_nose_ridge_def = {
+        1 = { name = nose_ridge_def_pos    range = { 0 1 }    }
+		5 = { name = nose_ridge_def_neg    range = { 0 1 }    }
+	}
+	
+	#Nostril
+	gene_bs_nose_nostril_height = {
+        1 = { name = nose_nostril_height_pos    range = { 0.25 0.6 }    }
+	}
+	gene_bs_nose_nostril_width = {
+        1 = { name = nose_nostril_width_pos    range = { 0 1 }    }
+		1 = { name = nose_nostril_width_neg    range = { 0 0.5 }    }
+	}
+	
+	###Ear
+	gene_bs_ear_angle = {
+        1 = { name = ear_angle_pos    range = { 0 0.5 } }
+	}
+	gene_bs_ear_inner_shape = {
+        1 = { name = ear_inner_shape_pos    range = { 0 1 } }
+    }
+	gene_bs_ear_bend = {
+        1 = { name = ear_upper_bend_pos    range = { 0 0.5 } }
+		1 = { name = ear_lower_bend_pos    range = { 0 0.5 } }
+		1 = { name = ear_both_bend_pos    range = { 0 0.5 } }
+    }
+	gene_bs_ear_outward = {
+        1 = { name = ear_outward_pos    range = { 1 1 } }
+    }
+	gene_bs_ear_size = {
+		1 = { name = ear_size_pos    range = { 1 1 } }
+    }
+	
+	###Forehead
+	gene_forehead_angle = {
+		1 = { name = forehead_angle_neg    range = { 0.4 0.7 }    }
+	}
+	gene_forehead_brow_height = {
+		1 = { name = forehead_brow_height_pos    range = { 0.3 0.75 }    }
+	}
+	gene_forehead_roundness = {
+		1 = { name = forehead_roundness_pos    range = { 0.5 0.85 }    }
+	}
+	gene_forehead_width = {
+		1 = { name = forehead_width_pos    range = { 0 0.65 }    }
+	}
+	gene_forehead_height = {
+		1 = { name = forehead_height_pos    range = { 0.75 1 }    }
+	}
+	gene_bs_forehead_brow_forward = {
+		1 = { name = forehead_brow_forward_pos    range = { 0.4 0.85 }    }
+	}
+	gene_bs_forehead_brow_curve = {
+		1 = { name = forehead_brow_curve_pos    range = { 0 1 }    }
+	}
+	gene_bs_forehead_brow_inner_height = {
+		1 = { name = forehead_brow_inner_height_pos    range = { 0 1 }    }
+	}
+	gene_bs_forehead_brow_outer_height = {
+		1 = { name = forehead_brow_outer_height_pos    range = { 0 1 }    }
+		1 = { name = forehead_brow_outer_height_neg    range = { 0 1 }    }
+	}
+	gene_bs_forehead_brow_width = {
+		1 = { name = forehead_brow_width_pos    range = { 0 1 }    }
+	}
+	
+	###Head
+	gene_head_height = {
+        1 = { name = head_height_neg    range = { 0 0.35 }    }
+    }
+	gene_head_width = {
+        1 = { name = head_width_neg    range = { 0.6 0.77 }    }
+    }
+	gene_head_profile = {
+        1 = { name = head_profile_neg    range = { 0.25 0.5 }    }
+    }
+	
+	#Head top
+	gene_head_top_height = {
+        1 = { name = head_top_height_neg    range = { 0 0.33  }    } # REMEMBER TO ADJUST RANGE!
+	}
+	gene_head_top_width = {
+        1 = { name = head_top_width_neg    range = { 1 1 }    } # REMEMBER TO ADJUST RANGE!
+	}
```

Comment:
**Remember remember...**

Reply:
Adjusted these in 6b389930b94a7a178d0bb1f07d2d6dcc0b76daab.